src/IECoreImage/DisplayDriverServer.cpp

```diff
@@ -70,6 +73,8 @@ IE_CORE_DEFINERUNTIMETYPED( DisplayDriverServer );
 namespace
 {
 
+static DisplayDriverServer::MergeMap g_mergeMap;
```

Comment:
I didn't spot this before, but are we happy that the merge map is shared across all servers like this, and there isn't a separate one per server? It seems like this forces the client to generate a _really_ unique ID, rather than only one that is unique within the server being connected to.

Reply:
In this context is the server something that is setup and running per every instance of Gaffer or would it be shared across all instance of Gaffer on a work station?

Reply:
Each instance of Gaffer will have its own server. Actually each instance will have a server for the Catalogue and then another server for each ShaderView (although we wouldn't expect those to be using the merging functionality).

But looking at DisplayDriverServer as a library component, which could be used anywhere, it might be better for the map to be per-server rather than static.

Reply:
Would making it non-static be enough or do you want `g_mergeMap` to be a member variable of the `DisplayDriverServer`?

Reply:
I think logically it should probably be a member.